api/src/main/java/io/opentelemetry/correlationcontext/CorrelationsContextUtils.java

```diff
@@ -77,21 +76,35 @@ public static CorrelationContext getValue() {
    * @return the value from the specified {@code Context}.
    * @since 0.1.0
    */
-  public static CorrelationContext getValue(Context context) {
-    return DIST_CONTEXT_KEY.get(context);
+  public static CorrelationContext getCorrelationContext(Context context) {
+    return CORR_CONTEXT_KEY.get(context);
+  }
+
+  /**
+   * Returns the value from the specified {@code Context}, falling back to a default, no-op {@link
+   * CorrelationContext}.
+   *
+   * @param context the specified {@code Context}.
+   * @return the value from the specified {@code Context}.
+   * @since 0.3.0
+   */
+  public static CorrelationContext getCorrelationContextWithDefault(Context context) {
+    CorrelationContext corrContext = CORR_CONTEXT_KEY.get(context);
+    return corrContext == null ? DEFAULT_VALUE : corrContext;
```

Comment:
Why not having `getCorrelationContext` never return null as it was before?

Reply:
I see the explanation in the description:
* I would suggest actually to have a `getCorrelationContextWithoutDefault` because the version with default will be the most used.
* I would suggest that we do not offer a `getCorrelationContextWithoutDefault` for the current Context because it will be used only by the propagators so limiting the API.

Reply:
Ok, we can do that. Will update the code. 

> I would suggest that we do not offer a getCorrelationContextWithoutDefault for the current Context because it will be used only by the propagators so limiting the API.

Sure, this works as well. We can always add it in the future if/as needed.

Reply:
Please update everywhere (span included).